packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -541,7 +541,9 @@ function createProjectionAndParams({
                 : `{ ${chainStr}: ${chainStr} }`;
 
             res.projection.push(
-                `${field.name}: apoc.cypher.runFirstColumn("${connection[0]} RETURN ${field.name}", ${runFirstColumnParams}, false)`
+                `${field.name}: apoc.cypher.runFirstColumn("${connection[0].replace(/"/g, '\\"')} RETURN ${
```

Comment:
Please, note that single quotes `'` needs to be escaped as well, replace should cover both:
```suggestion
                `${field.name}: apoc.cypher.runFirstColumn("${connection[0].replace(/("|')/g, "\\$1")} RETURN ${
```

Reply:
FYI, We already have a couple of utilities (`escapeQuery` and `wrapApocRun`) to take care of this issue in other places, however, these are not generic enough at the moment to be used here directly.